CONTRIBUTING.md

```diff
@@ -260,30 +257,9 @@ $ nix develop
 $ just integ-tests
 ```
 
-### Functional tests
-
-Functional tests are written with `bats`.
-They are located in the `./tests` folder.
-
-Functional tests should be used for testing:
-- Integration with external Flox services like FloxHub
-- Integration with language ecosystems
-
-Functional tests are not run on every commit on PRs.
-They are run in the merge queue or if a CI workflow is triggered manually on a
-branch from
-[Actions](https://github.com/flox/flox/actions).
-
-Functional tests can be run with `just`:
-
-```console
-$ nix develop
-$ just functional-tests
-```
-
 #### Mock catalog responses
 
-Mock catalog responses for use with integration and functional tests are generated by:
+Mock catalog responses for use with integration tests are generated by:
```

Comment:
Suggestion: I think grammatically this makes more sense to say "is generated by `mk_data` and stored in `test_data`" so it doesn't sound like `test_data` is generating mock responses.

Reply:
Bah, I resolved this but forgot to do it, sorry.